homeassistant/components/light/xiaomi_miio.py

```diff
@@ -112,23 +151,34 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
         from miio import PhilipsEyecare
         light = PhilipsEyecare(host, token)
         device = XiaomiPhilipsEyecareLamp(name, light, model)
+        devices.append(device)
+        hass.data[DATA_KEY][host] = device
+
+        device = XiaomiPhilipsEyecareLampAmbientLight(
```

Comment:
Please don't reuse the same local variable name within the same scope. That's confusing.